commands/removeCardCommand.js

```diff
@@ -1,36 +1,69 @@
 import db from "../db.js";
 
-const removeCardCommand = (message) => {
+const removeCardCommand = async (message) => {
   const cardToRemove = message.content.split(" ").slice(1).join(" ");
-  const ref = db.ref("users");
-  ref.once("value", (snapshot) => {
-    const userData = snapshot.val();
-    const userID = message.author.id;
-    const userInventory = userData[userID].inventory;
-
-    const cardIDToRemove = Object.keys(userInventory).find(
-      (key) =>
-        userInventory[key].name.toLowerCase() === cardToRemove.toLowerCase()
+  const ref = db.ref(`users/${message.author.id}`);
```

Comment:
for clarity it's helpful to specify what this ref is: `userRef` if its for a specific user, `usersRef` for the whole list of users, or `dbRef` or `ref` for the base level